package.json

```
@@ -26,7 +26,7 @@
     "debug": "~4.3.1",
     "mkdirp": "~0.5.0",
     "q": "~1.4.1",
-    "tar": "^4.4.19"
+    "tar": "^7.4.3"
```

I'm not a fan of `^` as it can introduce breaking changes
```suggestion
    "tar": "~7.4.3"
```